vf-react/src/js/components/layout/VFPagination.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
This Module handles the pagination at the bottom of grid pages. Right now it is only implemented in Ingredients page.